syntax = "proto3";
import "common.proto";
package protos;
import "google/protobuf/empty.proto";
// Single‚Äêmethod service
service HealthService {
   rpc Ping(google.protobuf.Empty) returns (Ack);
}
